scripts/orderIdValidation.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @param {import("@actions/core")} core
+ * @param {import("@actions/github").context} context
+ * @param {ReturnType<import("@actions/github").getOctokit>} github
+ */
```

Comment:
```suggestion
/**
 * @param {Object} params
 * @param {import("@actions/core")} params.core
 * @param {ReturnType<import("@actions/github").getOctokit>} params.github
 * @param {import("@actions/github").context} params.context
 */
```

Got to have the root when params is an object for correct typecheck

Reply:
works for me... still does not hurt though

Reply:
odd 🙃